services/QuillLMS/engines/evidence/app/controllers/evidence/feedback_controller.rb

```diff
@@ -4,75 +4,17 @@ module Evidence
   require 'json'
 
   class FeedbackController < ApiController
-    before_action :set_params, only: [:automl, :plagiarism, :regex, :spelling, :create]
+    before_action :set_params, only: [:create]
```

Comment:
nit: we no longer need the `only` specifier

Reply:
Agree that it's not strictly necessary, but I generally prefer to specify `only` for `before_actions` even if the intended list is all of them so someone doesn't run into issues if they add a new action later.  Could be convinced that this is overly verbose, though.